src/utils/git.rs

```diff
@@ -107,7 +107,10 @@ pub fn clone(mut url: String, branch: Option<String>, out: &std::path::Path) ->
         )
         .stdout_capture()
         .stderr_capture()
-        .run()
+        .start()
```

Comment:
Does this fix the weird coloring stuff? or is there a different reason you've made it run like this ?

Reply:
Did not mean to commit that, woops. Also yeah it doesn't fix it :(

Reply:
ah dang annoying